src/app/app.component.ts

```diff
@@ -1,10 +1,19 @@
 import { Component } from '@angular/core';
+import { IUser } from './models/user';
+import { AuthenticationService } from './services/authentication.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
-  title = 'Quiz-Front';
+  title = 'Quizzer';
+  currentUser: IUser;
+
+  constructor(private authenticatinService: AuthenticationService) {
+    this.authenticatinService.currentUser.subscribe(
+      (x) => (this.currentUser = x)
```

Comment:
I don't really like this formatting.
Firstly, add type for x, secondly, x is not a clear name for this variable. 
Let's do smth like that:
```
this.authenticatinService.currentUser.subscribe((user: UserType) => {
      this.currentUser = user;
    });
```